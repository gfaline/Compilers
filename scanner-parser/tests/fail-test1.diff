diff: tests/fail-test1.err: No such file or directory
