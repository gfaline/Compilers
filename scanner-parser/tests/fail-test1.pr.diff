diff: fail-test1.pr.err: No such file or directory
diff: testss/tests/fail-test1.pr.err: No such file or directory
