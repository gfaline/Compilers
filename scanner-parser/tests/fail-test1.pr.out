tests/fail-test1.pr
